src/flavors/__tests__/fixtures.ts

```diff
@@ -51,6 +52,16 @@ export function createFixtures(
     class Example {
         /* Create one property of each type that is required. */
 
+        @IsArray({
+            type: NestedRequiredExample,
+        })
+        requiredObjectArrayValue!: NestedRequiredExample[];
+
+        @IsArray({
+            type: Number,
```

Comment:
There's no nice way to skip the `type` argument here and trying to compose `@IsArray` with `IsNumber` is an engineering challenge, especially since I didn't plan for it.